lib/library.rb

```diff
@@ -0,0 +1,69 @@
+require 'yaml'
+require 'date'
+require 'pry'
+require './lib/person.rb'
+
+class Library
+
+attr_accessor :items
+
+def initialize
+  @items = YAML.load_file('./lib/data.yml')
+end
+
+def check_out(title, person)
+ case
+ when
+   if_not_available?(title, person) then 'Book is not available'
+ else
+   perform_checkout(title, person)
+ end
+end
+
+def see_list
+  items.each {|x| puts x }
+end
+
+def perform_checkin(title)
+  add_book(title, available: true, return_date: '')
+  # File.open('./lib/data.yml', 'w') { |f| f.write @items.to_yaml }
+end
+
+def perform_checkout(title, person)
+  change_status(title, available: false, return_date: Date.today + 30)
+  person.perform_borrow(title)
+  # File.open('./lib/data.yml', 'w') { |f| f.write @items.to_yaml }
```

Comment:
Make sure to delete unused code and these kind of comments before final submissions.